gcloud-eclipse-tools.launch

```diff
@@ -6,7 +6,7 @@
 <booleanAttribute key="automaticValidate" value="false"/>
 <stringAttribute key="bad_container_name" value="/trunk/gcloud-eclipse-tools-launcher"/>
 <stringAttribute key="bootstrap" value=""/>
-<stringAttribute key="checked" value="[NONE]"/>
+<stringAttribute key="checked" value="com.google.cloud.tools.eclipse.sdk"/>
```

Comment:
Revert the tracing change to the launch config.

Reply:
done

Reply:
This hasn't been pushed up?